helm_deploy/values-preprod.yaml

```diff
@@ -2,12 +2,9 @@
 # Per environment values which override defaults in hmpps-nomis-prisoner-api/values.yaml
 
 generic-service:
-  replicaCount: 4
```

Comment:
I assume 4 is défault anyway?

Reply:
yes, that value is in `values.yaml` too so doesn't need repeating here